doc/cc65.sgml

```diff
@@ -1119,6 +1119,21 @@ parameter with the <tt/#pragma/.
   remembered and output as a whole when translation is finished.
 
 
+<sect1><tt>#pragma message (&lt;message&gt;)</tt><label id="pragma-message"><p>
+
+  This pragma is used to display informational messages at compile-time.
+
+  Example:
+  <tscreen><verb>
+        #pragma message ("foo")
+  </verb></tscreen>
+
+  Results in the compiler outputting the following on the console:
```

Comment:
Did anybody notice the completely random selection of a line number example of 42 :)